### ISSUE TITLE
Cross-signing is hard to set up / wedged

### ISSUE THREAD

**Issue by Kegsay** [vector-im/riot-web#14347]
So I tried to set up cross-signing and had a Bad Time. First I went Settings -> Security -> Bootstrap where it asked me to enter in my account password followed by generating a new "Security Phrase" which I dutifully did. First problem I had was that the required password strength on riot-web is pretty insane, and I doubt I'll be able to remember the phrase as a result.

Regardless, I did it and then it asked for my account password again to set up cross-signing, so I did it. Then it prompted me again, and again, and again without telling me why it needs it so many damn times. I hit cancel and skipped setting things up, thinking I can go back in to continue. However, now I seem to have wedged my account - click on my own devices or clicking on "Bootstrap cross-signing and secret storage" prompts for my "Security Phrase" but then does nothing. Console logs:

> Got cross-signing update event for user @kegan:matrix.org but no new cross-signing information found!

Trying to set things up from RiotX were no better, I got a generic "Matrix Error" after entering my Security Phrase - which wasn't worded as a "Security Phrase" and made me think it was yet another password: using the same icon that riot-web uses for that prompt would've made it clearer it was referring to the same password.

So tl;dr I've somehow wedged my account by cancelling out of the tons of prompts for my account password.

Hitting "Reset cross-signing and secret storage" shows a pretty scary prompt "Deleting cross-signing keys is permanent. Anyone you have verified with will see security alerts. You almost certainly don't want to do this, unless you've lost every device you can cross-sign from." so I don't know what I should be doing instead.

**Comment by robintown** [comment]
@Kegsay Are you certain that the account password you entered was correct? I saw a user today encounter this same problem when they repeatedly entered the wrong password at that prompt—there was no indication that it was wrong, and it was impossible to dismiss.

**Comment by robintown** [comment]
@Kegsay Also, if you have never used cross signing on your account before, "Reset cross-signing and secret storage" shouldn't do anything bad.

**Comment by Kegsay** [comment]
I did end up resetting and got it working but yeah, I don't know what went wrong or how to fix it, so 🤷

**Comment by MurzNN** [comment]
I have the same problem with my @Murz:matrix.org account, now Element/develop every time suggest me to upgrade encryption, but after pressing "Upgrade" i see the error in console:
```
Got cross-signing update event for user @Murz:matrix.org but no new cross-signing information found!
```
I has use encrytion before, so making "Reset cross-signing and secret storage" will be not good solution?

**Comment by richvdh** [comment]
> I did end up resetting and got it working but yeah, I don't know what went wrong or how to fix it, so 🤷

I don't think we can do much with this.
